### PR TITLE
Qt: Add Vulkan to graphics backend description

### PR DESCRIPTION
**Pull request by 8times9**

### CODE REVIEW

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Comment by legoj15** [comment]
I could've sworn there was another PR like this that was merged already...….

**Comment by 8times9** [comment]
@legoj15 I had a pull request for this a few weeks ago, but as I'm still new to git I made some mistakes with the commits and decided to start over.

File: Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp [hunk 1]
```diff
@@ -231,12 +231,12 @@ void GeneralWidget::AddDescriptions()
...
 // We need QObject::tr
 #if defined(_WIN32)
-  static const char TR_BACKEND_DESCRIPTION[] =
-      QT_TR_NOOP("Selects what graphics API to use internally.\nThe software renderer is extremely "
-                 "slow and only useful for debugging, so you'll want to use either Direct3D or "
-                 "OpenGL. Different games and different GPUs will behave differently on each "
-                 "backend, so for the best emulation experience it's recommended to try both and "
-                 "choose the one that's less problematic.\n\nIf unsure, select OpenGL.");
+  static const char TR_BACKEND_DESCRIPTION[] = QT_TR_NOOP(
+      "Selects what graphics API to use internally.\n\nThe software renderer is extremely "
```
Comment (MayImilae) [inline Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: I'm not a coder and can't really give code review, but why two new lines (\n) here?

Comment (KAMiKAZOW) [inline Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: "what" not "which"?

File: Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp [hunk 2]
```diff
@@ -231,12 +231,12 @@ void GeneralWidget::AddDescriptions()
...
+      "slow and only useful for debugging, so you'll want to use either OpenGL, Direct3D, "
+      "or Vulkan. Different games and different GPUs will behave differently on each "
+      "backend, so for the best emulation experience it's recommended to try each and "
```
Comment (KAMiKAZOW) [inline Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: AFAIK all major HIGs recommend not to use casual language like "it's" and suggest "it is" instead. Same with "you'll".

File: Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp (hunk 1 above)
Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: I honestly thought it looked better, line breaks without a space in between the separated lines look weird to me. 

If you don’t like it I can revert it.

File: Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp (hunk 2 above)
Comment (8times9) [reply Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: Yeah, IIRC a lot of descriptions in Dolphin use contractions :/

I’ll fix it here

**Pull request merged**